cpp/src/Slice/Parser.h

```diff
@@ -363,10 +363,14 @@ namespace Slice
     {
     public:
         [[nodiscard]] ContainerPtr container() const;
+
         [[nodiscard]] std::string name() const;
         [[nodiscard]] std::string scoped() const;
         [[nodiscard]] std::string scope() const;
-        [[nodiscard]] std::string flattenedScope() const;
+        [[nodiscard]] std::string mappedName() const;
```

Comment:
It would be helpful to add doc-comments.

Reply:
I added some short doc-comments to each of these 6 functions.